back/engines/commercial/id_clave_unica/spec/requests/clave_unica_verification_spec.rb

```diff
@@ -248,12 +248,14 @@ def expect_to_create_verified_user(user)
         headers = { 'Authorization' => "Bearer #{token}" }
         post '/web_api/v1/user/resend_code', params: { new_email: 'newcoolemail@example.org' }, headers: headers
         expect(response).to have_http_status(:ok)
-        expect(user.reload).to have_attributes({ email: 'newcoolemail@example.org' })
+        expect(user.reload).to have_attributes({ new_email: 'newcoolemail@example.org' })
```

Comment:
It seems previously `email` (not `new_email`) was set because `active?` returned `false` in `User#reset_email!`. If we don't need this logic anymore, this change in this spec should be fine. 

I also tested it manually and it works fine.

Reply:
That could be it indeed! I also verified changing the email of an inactive user and worked fine for me as well. Thanks for looking into this!